sbncode/BeamSpillInfoRetriever/NuMIRetriever/NuMIRetriever_module.cc

```diff
@@ -109,6 +109,13 @@ void sbn::NuMIRetriever::produce(art::Event &e)
   e.getByLabel(raw_data_label_, "ICARUSTriggerV3", raw_data_ptr);
   auto const & raw_data = (*raw_data_ptr);
 
+  // NOTE: Really we should skip the first event of each trigger type, so let's make this look at that too...
+  if ( raw_data.size()==0 ) return;
```

Comment:
I can't believe I missed one of my favourite pet peeves!
```suggestion
  if ( raw_data.empty() ) return;
```